assets/stylesheets/screens/_oauth_client.scss

```diff
@@ -0,0 +1,5 @@
+.o-auth-client-controller {
```

Comment:
I think the file name should be `_oauth-client.scss `

Reply:
@junan nice catch, fixed in [d1e5d2b](https://github.com/carryall/go-google-scraper-challenge/pull/28/commits/d1e5d2ba660ea653d245f62ff6c17267fe0fc8a8)